docs/contribution-guide.md

```diff
@@ -46,7 +46,7 @@ Even if you have push rights on the [gitthermal/thermal](https://github.com/gitt
 
 ## Pull Requests
 
-To enable us to quickly review and accept your pull requests, always create one pull request per issue and [link the issue in the pull request](https://github.com/blog/957-introducing-issue-mentions). Never merge multiple requests in one unless they have the same root cause. Be sure to follow our [Coding Guidelines](/contribute/coding-guidelines/) and keep code changes as small as possible. Avoid pure formatting changes to code that has not been modified otherwise. Pull requests should contain [tests](https://github.com/gitthermal/thermal/issues/26) whenever possible.
+To enable us to quickly review and accept your pull requests, always create one pull request per issue and [link the issue in the pull request](https://github.com/blog/957-introducing-issue-mentions). Never merge multiple requests in one unless they have the same root cause. Be sure to follow our Coding Guidelines (still work in progress 😅) and keep code changes as small as possible. Avoid pure formatting changes to code that has not been modified otherwise. Pull requests should contain [tests](https://github.com/gitthermal/thermal/issues/26) whenever possible.
```

Comment:
```suggestion
To enable us to quickly review and accept your pull requests, always create one pull request per issue and [link the issue in the pull request](https://github.com/blog/957-introducing-issue-mentions). Never merge multiple requests in one unless they have the same root cause. Be sure to follow our Coding Guidelines _(still work in progress 😅)_ and keep code changes as small as possible. Avoid pure formatting changes to code that has not been modified otherwise. Pull requests should contain [tests](https://github.com/gitthermal/thermal/issues/26) whenever possible.
```

Reply:
Just a minor change and we are good to merge this PR.